elastica/boundary_conditions.py

```diff
@@ -258,28 +260,31 @@ class OneEndFixedRod(OneEndFixedBC):
     )
 
 
-class FixedConstraint(ConstraintBase):
+class ConfigurableFixedConstraint(ConstraintBase):
```

Comment:
change name maybe to 
```suggestion
class ConfigurableConstraint(ConstraintBase):
```
and if changed, propagate everywhere.

Reply:
Changed in https://github.com/GazzolaLab/PyElastica/pull/143/commits/4a9c71e4faa98ca2e35e461a9b3f79c1edcfa2d7